singularity-eos/eos/eos_base.hpp

```diff
@@ -21,6 +21,16 @@
 #include <ports-of-call/portability.hpp>
 #include <ports-of-call/portable_errors.hpp>
 
+#if 0
+//#ifdef SINGULARITY_USE_SPINER_WITH_HDF5
+#include <spiner/databox.hpp>
+namespace singularity {
+
+using DataBox = Spiner::DataBox<Real, std::true_type>;
+
+} // namespace singularity
+#endif // SINGULARITY_USE_SPINER_WITH_HDF5
```

Comment:
```suggestion
#endif // 0
```

Reply:
?

Reply:
I can do this, but if you don't want the comment then just get rid of the comment? (why `// 0`?)

Reply:
Yeah I'm a bit confused as well. I appreciate comments like this at `#endif` statements

Reply:
Because this if case is a comment. If you look above, the whole block of code is commented out with `#if 0`

Reply:
I think we should remove it from the PR if it's universally removed by the preprocessor.

Reply:
Ah sorry for the confusion. Yes I agree then

Reply:
Removed the dead code